### DISCUSSION TITLE
Velero Upgrade Crds Container Back-off

### DISCUSSION THREAD

**Discussion by allwin-winfred** [Sunbird-Obsrv/Community#151] [Installation] [answered]
Hello,
I'm running Velero on EKS (1.28) while installing Velero using the helm chart I'm facing an Issue in the Pod back-off restart container,
also, I'm deploying this chart using Terraform,
In the terraform console facing this error,
│ Error: release velero failed, and has been uninstalled due to atomic being set: failed pre-install: job failed: BackoffLimitExceeded

Then I inspected the Velero pod while installing it says below the error,

**kubectl logs -n velero velero-upgrade-crds-lf5j9

Defaulted container "velero" out of: velero, kubectl (init)
/tmp/sh: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/sh)**

**Comment by sowmya-dixit** [comment, marked as answer]
@allwin-winfred 
We have noticed this issue in recent versions and we are doing the patch in upcoming release. As a workaround please use this approach of adding below configurations under Verelo helm chart **values.yaml**  - [https://github.com/Sunbird-Obsrv/obsrv-automation/tree/main/terraform/modules/helm/velero](https://github.com/Sunbird-Obsrv/obsrv-automation/tree/main/terraform/modules/helm/velero)
```
kubectl:
  image:
    repository: docker.io/bitnami/kubectl
    tag: 1.26.14-debian-11-r6
```
**Note**: Also please add same configurations under **aws-velero.tfpl.yaml** and **gcp-velero.tfpl.yaml**.

**Reply by allwin-winfred** [reply]
@sowmya-dixit 
Now it's working fine, Thank You.
